src/applications/edu-benefits/1990/config/form.js

```diff
@@ -17,6 +17,8 @@ import preSubmitInfo from 'platform/forms/preSubmitInfo';
 import { VA_FORM_IDS } from 'platform/forms/constants';
 import currentOrPastMonthYearUI from 'platform/forms-system/src/js/definitions/currentOrPastMonthYear';
 import yearUI from 'platform/forms-system/src/js/definitions/year';
+import { validateBooleanGroup } from 'platform/forms-system/src/js/validation';
+import dateUI from 'platform/forms-system/src/js/definitions/date';
```

Comment:
Did these end up in here by accident?

Reply:
Yeah I was going to say the same thing. I assume that's why the other team got pinged as a reviewer as well. This almost looks like some linting fix that happened...because all this change shows is moving those two lines of code up 20 lines.

Reply:
File-commit reverted.